### PR TITLE
[OpenVINO-EP] Enabling VAD-F in OpenVINO Execution Provider

### PR DESCRIPTION
**Pull request by suryasidd**
**Description**: Enabling VAD-F accelerator in OpenVINO EP

**Motivation and Context**
- This is required for running inference on Intel Arria 10 FPGAs

### CODE REVIEW

File: tools/ci_build/build.py [hunk 1]
```diff
@@ -875,6 +876,10 @@ def main():
     if args.build_csharp:
         args.build_shared_lib = True
 
+    if args.use_openvino == "VAD-F_FP32":
+        args.test = False
+        args.skip_onnx_tests = True
```
Comment (jywu-msft) [inline tools/ci_build/build.py]: setting args.tests = False should be sufficient. don't need to also set skip_onnx_tests.
can you also a comment as to why tests are disabled for VAD-F_FP32

Comment (suryasidd) [reply tools/ci_build/build.py]: Cool. Will add this

**Pull request merged**